azure-kusto-data/azure/kusto/data/_version.py

```diff
@@ -1,3 +1,3 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT License
-VERSION = "1.0.2"
+VERSION = "2.0.0"
```

Comment:
I think you should revert instead.
Let's discuss this with @ArielYehezkely

Reply:
And please don't bump version in the same PR.

Reply:
@yogilad I'm reverting, and will keep this as a separate PR.
I'm watching/starred this repo, so I'll be notified of all PRs. When I see something that introduces a different breaking change, I'll merge this change as well.

@ArielYehezkely Is the rationale that a bump in version is irreversible, while would be relevant if the PR with the change is reverted?

Reply:
1. Correct you don't want to revert this unintentionally.
2. We don't release new version for every change, so we can have few commits between releases and its easier to track the version bumping as a separate commit. (v.1.0.0 .. A-B-C .. v1.0.1) we create a tag for each release and its nicer to attach it to the relevant commit and not to the code change itself.